ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/CommandRunner.java

```diff
@@ -91,6 +81,20 @@ public void close() {
     commandStore.close();
   }
 
+  public void fetchAndRunCommands() {
+    ConsumerRecords<CommandId, Command> records = commandStore.getNewCommands();
+    log.debug("Found {} new writes to command topic", records.count());
```

Comment:
Maybe this message should be `trace` level? It has potential to be quite spammy

Reply:
Good point, changed it to trace.